demo/rn/src/screens/veronoi-screen.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+import { ScrollView } from "react-native";
+import viewStyles from "../styles/view-styles";
+import { VictoryTheme, VictoryVoronoi, VictoryChart } from "victory-native";
+
+export const VeronoiScreen: React.FC = () => {
```

Comment:
NIT: "voronoi" not "veronoi"

Reply:
😆 good catch. I've never seen that word before, and I let my fingers take the wheel. I misspelled that multiple times (I was wondering why my editor spelling suggestions were mad about "veronoi" lol). Just patched those.